<launch>
   <arg name="device_name"                default="/dev/ttyUSB0"/>
   <arg name="baud_rate"                  default="3000000"/>
   <arg name="latency_timer"              default="1"/>
   <arg name="scan_range"                 default="10"/>

   <arg name="profile_velocity"           default="75"/>
   <arg name="profile_acceleration"       default="75"/>

   <param name="device_name"              value="$(arg device_name)"/>
   <param name="baud_rate"                value="$(arg baud_rate)"/>
   <param name="latency_timer"            value="$(arg latency_timer)"/>

   <param name="scan_range"               value="$(arg scan_range)"/>

   <param name="profile_velocity"         value="$(arg profile_velocity)"/>
   <param name="profile_acceleration"     value="$(arg profile_acceleration)"/>
   
   <node name="position_control" pkg="dynamixel_workbench_controllers" type="position_control" required="true" output="screen"/>
   
   
   <node name="dynamixel_action_server" pkg="open_manipulator_core" type="trajectory_server" required="true" output="screen" />
   <node name="gripper_commander" pkg="open_manipulator_core" type="gripper_commander" required="true" output="screen" /> 
 </launch>
